how-to/business-logic/how-to-get-result-status-request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to Get the Result Status of the Request
+description: An example to perform a check of the result request status
+type: how-to
+page_title: How to Get the Result Status of the Request in the {{site.bs-bl}} Layer?
+slug: bl-get-request-result
+position:
+tags: backendservices, business-logic, cloud-code
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+**Applicable only to {{site.bl-data}}**. Frequently you need to check if the data operation completed with a success or an error in order to trigger up additional business logic. 
+
+## Solution
+
+You can check how the data operation completed **only** in the {{site.bl-data}} `after*` event handlers using the `response` parameter.

```

Comment:
**only**: As per style guide, this kind of emphasis is realized using italic, not bold.